tests/end_to_end/netsh_test_helper.h

```diff
@@ -41,3 +41,6 @@ _run_netsh_command(
     _In_opt_z_ const wchar_t* arg2,
     _In_opt_z_ const wchar_t* arg3,
     _Out_ int* result);
+
+std::string
+run_netsh_command_va_arg(_In_ FN_HANDLE_CMD* command, _Out_ int* result, int argc, ...);
```

Comment:
"va_arg" is probably not quite right since the meaning is slightly different from the va_arg standard api which accepts a va_list as an input parameter. 
```suggestion
run_netsh_command_with_args(_In_ FN_HANDLE_CMD* command, _Out_ int* result, int argc, ...);
```